src/spec/spec.cr

```diff
@@ -209,7 +209,7 @@ OptionParser.parse! do |opts|
     exit
   end
   opts.on("-v", "--verbose", "verbose output") do
-    Spec.formatter = Spec::VerboseFormatter.new
+    Spec.formatters << Spec::VerboseFormatter.new
```

Comment:
@asterite there is a problem here, if we just push a new formatter I believe it is going to use the both at the same time and create a unexpected behaviour. We have 2 options, use only the `VerboseFormatter` or keep this behaviour.